elfpy/agents/policies/single_lp.py

```diff
@@ -38,3 +39,33 @@ def action(self, _market: hyperdrive_market.Market) -> "list[types.Trade]":
                 )
             )
         return action_list
+
+
+class SingleLpAgent(agent.AgentFP):
+    """simple LP that only has one LP open at a time"""
+
+    def __init__(self, wallet_address, budget=FixedPoint("1000.0")):
+        """call basic policy init then add custom stuff"""
+        self.amount_to_lp = FixedPoint("100.0")
+        super().__init__(wallet_address, budget)
+
+    def action(self, _market: hyperdrive_market.Market) -> "list[types.Trade]":
```

Comment:
if you add `from __future__ import annotations` then you can remove the quotes around the return type.